modules/core/src/main/java/org/apache/ignite/internal/processors/ru/IgniteRollingUpgradeStatus.java

```diff
@@ -108,6 +107,7 @@ public IgniteRollingUpgradeStatus(
         out.writeBoolean(forcedModeEnabled);
         out.writeObject(initVer);
         out.writeObject(updateVer);
+        out.writeObject(new HashSet<>());
```

Comment:
This will work, but it's better to override getProtocolVersion() and use it in the code like in other data transfer objects. It will raise less questions this way.